tests/test_startup.py

```diff
@@ -245,26 +245,39 @@ def _mock_folder_listing(self):
         else:
             yield
 
-    def _get_plugin_environment(self, dcc_path):
+    def _join_existing_environment(self, dcc_path, startup_path):
+        """
+        Returns the pre-existing environment paths for dcc_path, os.pathsep joined with the startup_paths
+        """
+        startup_path = [startup_path]
+        existing_paths = os.environ.get(dcc_path)
+        # combine the existing dcc_path paths, with the sgtk startup_paths only if there are preexisting paths
+        startup_paths = existing_paths.split(os.pathsep) + startup_path if existing_paths else startup_path
+        # now combine the list of paths with the os specific separator
+        return os.pathsep.join(startup_paths)
+
+    def _get_plugin_environment(self, dcc_path, ):
```

Comment:
if possible, seems we should add a test where the we already have a NUKE_PATH set to test the order in which the new NUKE_PATH is being assembled.